ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml

```diff
@@ -57,6 +57,30 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: optional-operators-ci-aws
+- as: operator-e2e-periodic-slack
+  cron: 0 4 * * *
```

Comment:
Sorry, I might be wrong on this, but is this running every hour on the 4th min? Is that what we want that seems like a lot of runs?

Reply:
This runs at 4am UTC = 12am EST.  "*/4 * * * *" runs at every 4th minute, and "0 */4 * * *" runs at every 4th hour.

Reply:
Thanks, had to go do a refresh on cron syntax!